app/models/founder.rb

```diff
@@ -104,7 +104,7 @@ def dropped_out?
   end
 
   def latest_submissions
-    timeline_events.where(timeline_event_owners: { latest: true })
+    timeline_events.live.where(timeline_event_owners: { latest: true })
```

Comment:
All usages of `timeline_events._something_` should be reached. 
Example: 
![image](https://user-images.githubusercontent.com/14979190/153197263-e60f83af-b21c-4fe3-b3e6-0a8c61f3703a.png)